### PR TITLE
Fixes #51 - React setup with Webpack and Babel

### LINKED ISSUE #51: setting up react structure
I would break down the home page to components and put them in the component folder. This way @Jojokats  and I we can choose which component to work on.  Also from the project structure, I recommend having index.js in the src folder and components as a separated folder inside src. 

I do not know where to put index.html yet. I will update this issue when I found out.
I make a PR with my changes. Any ideas, thoughts are welcome.

### PR DESCRIPTION
**Pull request by ZJaf**
This PR has some of our my updates in some components.

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,24 @@
+0 info it worked if it ends with ok
```
Comment (RichardLitt) [inline npm-debug.log]: No need to commit this file. :)

**Comment by RichardLitt** [comment]
Is this ready to be reviewed? :)

**Comment by ZJaf** [comment]
We decided to go with boilerplate. I stopped working on this. Shall we close this PR?

**Comment by RichardLitt** [comment]
Sure. Go ahead. :)

**Pull request closed without merging**